assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/IdeaActivityButton.js

```diff
@@ -56,7 +56,7 @@ export default function IdeaActivityButton( {
 	// Hide action buttons if the user has view-only version of the dashboard.
 	const viewOnlyDashboard = useViewOnly();
 	if ( viewOnlyDashboard && activity !== IDEA_HUB_BUTTON_VIEW ) {
-		return null;
+		return <Null />;
```

Comment:
This is not really the way the `Null` component is intended to be used; we shouldn't use it here just because it's imported.

`Null` is essentially a null object that allows us to use an `ElementType` in a polymorphic way so that it can always be used as `<Component />` as it is done below with `ActivityIcon` 🙂

Reply:
Thanks @aaemnnosttv That's good to know! 😁 
@hussain-t All good on my side then!